build_tools/github/build_source.sh

```diff
@@ -10,8 +10,3 @@ python -m pip install pytest pandas
 python setup.py sdist
 python -m pip install dist/*.tar.gz
 python setup.py build_ext -i
```

Comment:
While you are doing this, I think we should actually test the installed source distribution from site-packages instead of running the test from the source tree with the inplace built extensions.

```suggestion
```

Reply:
You are absolutely right.